### ISSUE TITLE
[TASK] Add notice about remaining time for `Claimable Vested Balance` row

### ISSUE THREAD

**Issue by AtelyPham** [webb-tools/webb-dapp#2306]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Upon clicking the `Vest` button on the Account page of the Tangle dApp, the `Claimable Vested Balance` continues to display in the balances table with a negligible amount. Furthermore, even after refreshing the page, the `Vest` button remains clickable.

### Expected Behavior

After users vest their funds, the `Claimable Vested Balance` should not be displayed, and the `Vest` button should become non-clickable.

### Steps To Reproduce

1. Navigate to the Account page on the Tangle dApp.
2. Click the `Vest` button.
3. Complete the transaction approval process.
4. Refresh the page. Observe that the `Claimable Vested Balance` still shows a minimal amount, and the `Vest` button remains clickable.

### Screenshot

https://github.com/webb-tools/webb-dapp/assets/60747384/043e7155-064c-40f3-8068-c74e19fa1fb2

**Comment by yurixander** [comment]
@AtelyPham Could you show me the vesting schedule details by expanding the locked balance details section? I think that the reason why the vest button keeps getting re-activated is because tTNT keeps getting vested at each block. So when you click the `vest` button, it doesn't claim *all* of the vesting balance, but rather all that has been vested thus far.

Until the vesting schedule ends, the button should remain visible and clickable in case that the user wishes to release the vested funds up to their active block. `Claimable vested balance` is what has unlocked from vesting thus far, so it doesn't represent the **total** vesting schedule amount.

In other words, so far I believe this is correct behavior, but looking at the vesting schedule details should help us confirm that.

**Comment by yurixander** [comment]
One other thing that I am curious about, I noticed that the active network is mainnet; How were you able to create a vesting schedule on mainnet, or was it an account that had it from genesis, and it started unlocking now? I thought that creating vesting schedules in mainnet should have been disabled...

**Comment by AtelyPham** [comment]
Yes, I've used this account to claim the airdrop for testing purposes on the mainnet. I understand what you're saying about a partial balance being unlocked with each block since the vesting time started. This means we can claim a small portion of the fund now or wait until the full amount is unlocked to save on fees.

We could provide more information, such as the remaining time until the full amount is unlocked, the rate at which tokens will be unlocked per block, etc., to make it clearer. What do you think?

![CleanShot 2024-05-12 at 21 38 35@2x](https://github.com/webb-tools/webb-dapp/assets/60747384/91624ba9-9c2b-4ae3-bf2c-36fff4fc8ea3)

**Comment by yurixander** [comment]
> We could provide more information, such as the remaining time until the full amount is unlocked, the rate at which tokens will be unlocked per block, etc., to make it clearer. What do you think?

Try hovering over the blue dots!

**Comment by yurixander** [comment]
Is this good to close? @AtelyPham

**Comment by AtelyPham** [comment]
> We could provide more information, such as the remaining time until the full amount is unlocked, the rate at which tokens will be unlocked per block, etc., to make it clearer. What do you think?

Oh, sorry, I missed your response. The information I meant to add is for the `Claimable Vested Balance`.

**Comment by yurixander** [comment]
We currently have the following information shown on vesting schedules:
* Their remaining time.
* The block at which they unlock fully.
* How many blocks left until then.
* Vested amount, and the amount remaining to vest.

![image](https://github.com/webb-tools/webb-dapp/assets/101931215/3841641a-b9bd-4380-89d8-4ff5510a7430)
![image](https://github.com/webb-tools/webb-dapp/assets/101931215/cfcb0070-2c8a-42aa-80e9-f07a033328ec)

The change I would be making here would be:
- Adding a notice for users to be aware that they'll only be claiming a partial amount of their vested balance,
- and also how long until all vested balance can be claimed.

**Comment by yurixander** [comment]
@AtelyPham Thoughts on this?

> Adding a notice for users to be aware that they'll only be claiming a partial amount of their vested balance,

![image](https://github.com/webb-tools/webb-dapp/assets/101931215/b7705ad4-e526-4129-a9a6-404feca4abf2)

> and also how long until all vested balance can be claimed.

![image](https://github.com/webb-tools/webb-dapp/assets/101931215/17f2ef84-8f62-4d90-8911-0f13af59637b)
